nvm.sh

```diff
@@ -2349,17 +2368,17 @@ nvm() {
   COMMAND="${1-}"
   shift
 
-  # initialize local variables
+  # Initialize local variables
```

Comment:
```suggestion
  # initialize local variables
```
no need to capitalize this.

Reply:
Sure, I will commit the suggestion.